### ISSUE TITLE
label files for pretrained slim model?

### ISSUE THREAD

**Issue by flyfj** [tensorflow/models#501]
## pretrained models

Anyone knows how to get the corresponding labels for pretrained models, e.g. inception v3? only checkpoint file is provided.

**Comment by tatatodd** [comment]
@flyfj have you seen this documentation:
https://github.com/tensorflow/models/tree/master/slim#downloading-and-converting-to-tfrecord-format
